recipes/txl/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "txl" %}
+{% set version = "0.2.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ceb02e2e00b46a9493a5bd0c302aecd5590f85c2122a41aa825aedd335b79854
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+    - hatchling
+  run:
+    - python >=3.10
+    - asphalt >=4.12.0,<5.0.0
+    - textual >=0.71.0,<0.72.0
+    - tree_sitter
+    - tree_sitter_languages
+
+test:
```

Comment:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

Reply:
Unfortunately this will fail because Asphalt's version appears as `0.0.0`, see https://github.com/asphalt-framework/asphalt/issues/73.

Reply:
@xhochy I reverted your changes in https://github.com/conda-forge/staged-recipes/pull/26869/commits/ec389865b59adb16837b205b02043cd6ba30c66b, as I said above it [fails](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=971584&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823).